scilpy/utils/metrics_tools.py

```diff
@@ -249,18 +250,33 @@ def plot_metrics_stats(mean, std, title=None, xlabel=None,
     if figlabel is not None:
         fig.set_label(figlabel)
 
+    if means.ndim > 1:
+        mean = np.average(means, axis=1)
+        std = np.average(stds, axis=1) + np.std(means, axis=1)
```

Comment:
Its actually mathematically bad, a researcher would have to do much better for the analysis.
But for plotting it is fine, its because each subject has a mean and std, then at the population level we were showing the variation of mean only discarding the subject std. I believe this gave a wrong impression of stability and was kind of like lying to the user by hiding variation.

So to approx this, I add up how much the average of subject variation on top of their average intra-subject variation.
@mdesco Do you have a better solution, at least just for visualization?